test/shallowDiff.js

```diff
@@ -0,0 +1,39 @@
+// import should from 'should';
+
+import shallowDiff from '../src/wrap-store/shallowDiff';
+
+describe('#shallowDiff()', () => {
+  it('should return an object containing updated fields', () => {
+    const old = { a: 1 };
+    const latest = { a: 2, b: 3 };
+    const diff = shallowDiff(old, latest);
+
+    diff.length.should.eql(2);
+    diff.should.eql([
+      {
+        key: 'a',
+        value: 2,
+        change: 'updated',
```

Comment:
Loving these tests. When you extract out that constants file, could you reference the constants here as well?